lib/PuppeteerSharp/ChromeTargetManager.cs

```diff
@@ -313,7 +329,7 @@ private async Task OnAttachedToTarget(object sender, TargetAttachedToTargetRespo
                 }
             }
 
-            await _targetDiscoveryCompletionSource.Task.ConfigureAwait(false);
+            await EnsureTargetsIdsForInit().ConfigureAwait(false);
```

Comment:
This is way better than looking for the variable every time. Thanks! Let's wait for the green!